TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/EditInduction/InductionChangeReason.cshtml.cs

```diff
@@ -25,8 +25,15 @@ public InductionChangeReasonModel(TrsLinkGenerator linkGenerator) : base(linkGen
     {
     }
 
-    public void OnGet()
+    public Task OnGetAsync()
     {
+        return JourneyInstance!.UpdateStateAsync(state =>
```

Comment:
Shouldn't be doing updates in a Get method .. and initial state setting should be done in `EnsureInitializedAsync`

Also should probably be changing this page as part of a separate ticket for evidence / change reason etc